hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/strategy/PartitionAwareClusteringPlanStrategy.java

```diff
@@ -55,7 +59,7 @@ public PartitionAwareClusteringPlanStrategy(HoodieTable table, HoodieEngineConte
   /**
    * Create Clustering group based on files eligible for clustering in the partition.
    */
-  protected Stream<HoodieClusteringGroup> buildClusteringGroupsForPartition(String partitionPath, List<FileSlice> fileSlices) {
+  protected Pair<Stream<HoodieClusteringGroup>, Boolean> buildClusteringGroupsForPartition(String partitionPath, List<FileSlice> fileSlices) {
```

Comment:
Can we add some doc on the boolean flag.

Reply:
added.
```
  /**
   * Create Clustering group based on files eligible for clustering in the partition.
   * return stream of HoodieClusteringGroup and boolean partial Scheduled indicating whether all given fileSlices in the current partition have been processed.
   * For example, if some file slices will not be processed due to writeConfig.getClusteringMaxNumGroups(), then return false
   */
```